code/drasil-lang/Language/Drasil/Chunk/Constrained.hs

```diff
@@ -60,13 +87,16 @@ cvc :: String -> NP -> Symbol -> Space -> [Constraint] -> Maybe Expr -> Constrai
 cvc i des sym space = ConstrainedChunk (qw (vc i des sym space))
 
 -- | Creates a new ConstrainedChunk from either a 'ConstrainedChunk', 'ConstrConcept', 'UncertainChunk', or an 'UncertQ'.
-cnstrw :: (Quantity c, Constrained c, HasReasVal c, MayHaveUnit c) => c -> ConstrainedChunk
-cnstrw c = ConstrainedChunk (qw c) (c ^. constraints) (c ^. reasVal)
+cnstrw :: (Quantity c, Constrained c, HasReasVal c, MayHaveUnit c) => c -> ConstrainedChunk  
+cnstrw c = ConstrainedChunk   (qw c) (c ^. constraints) (c ^. reasVal)
+
+
+
 
 -- | ConstrConcepts are conceptual symbolic quantities ('DefinedQuantityDict')
 -- with 'Constraint's and maybe a reasonable value (no units!).
 data ConstrConcept = ConstrConcept { _defq :: DefinedQuantityDict
-                                   , _constr' :: [Constraint]
+                                   , _constr'' :: [Constraint]
```

Comment:
why the extra \' ?

Reply:
Since constrained chunk was still present i had to change the `_constr` in `ConstrainedQDef` to `_constr'` and this to `_constr''`

After all the new chunk types are in all the ' should be uniform then

Reply:
Ah, indeed, makes sense.